.circleci/conditional_config.yml

```diff
@@ -203,6 +204,12 @@ jobs:
                 command: |
                   cd src/api
                   bundle exec rake dev:lint:apidocs
+      - run:
+          name: Run migration tests
+          command: |
+            cd src/api
```

Comment:
We could use the `working_directory` key, instead of adding a `cd ...` line in the command. See https://circleci.com/docs/configuration-reference/#run

Reply:
You applied it.